### ISSUE TITLE
Optifine prevents player render from shrinking or growing

### ISSUE THREAD

**Issue by CammiePone** [sp614x/optifine#2275]
Somewhere in Optifine, there's something that prevents the Forge player render event from working properly, leading to people posting bug reports to the wrong mod, such as this (https://github.com/CammiePone/Resize-Potion/issues/10).

As Optifine worked prior to the switch to using [ArtemisLib](https://github.com/CammiePone/ArtemisLib), it's likely to do with something in there, but is definitely still something Optifine is doing (maybe with Entity Attributes, or Capabilities?), as it still works with other mods, even ones that replace the model with a custom one.

**Comment by LokeYourLord** [comment]
An update would be nice.

**Comment by sp614x** [comment]
Is this a conflict with ResizePotion or is ArtemisLib also included?
What is the minimal amount of mods installed that reproduce the problem?

**Comment by CammiePone** [comment]
Let me double check real quick (as soon as my computer stops being frozen) if I accidentally fixed it with an update to the rendering code a few days ago that was targeting the issue of the render not matching the hitbox. I doubt it, but it might have done something.

**Comment by LokeYourLord** [comment]
Minimal amounts of mods is just Forge, Optifine, ArtemisLib and ResizePotion, cannot be any more minimal than so. ArtemisLib is needed for ResizePotion so I am afraid it is like CammiePone said, it is an issue with ArtemisLib.

**Comment by CammiePone** [comment]
Alright, first tested with:
Forge 1.12.2-14.23.5.2824
Optifine 1.12.2 HD U E3
ArtemisLib 1.0.4
Resizing Potion 2.1

With the latest version of ArtemisLib and Resizing Potion, it seems something I did inadvertently fixed this issue. Though Optifine is in the extracted JAR form, as that's the only way I could get it working in the Twitch launcher. I'm not sure if that's how it's meant to run alongside Forge or not.

Second test was with:
Forge 1.12.2-14.23.5.2824
Optifine 1.12.2 HD U E3
ArtemisLib 1.0.4
Squash 1.0.0

Squash is another mod that uses ArtemisLib for resizing entities. Everything else above still applies. Optifine is being used in its extracted JAR form, and it appears that what I did, while really shouldn't have made a difference, seems to make a difference.
